octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/search/SearchEditor.java

```diff
@@ -105,6 +109,13 @@ public void createPartControl(Composite parent) {
         container = new StackLayoutComposite(parent, SWT.NONE);
         container.setBackground(SWTResourceManager.getColor(SWT.COLOR_TRANSPARENT));
         container.setForeground(foregroundColor);
+        
+        Set<Entity> searchEntityTypesCopy = new LinkedHashSet<>(searchEntityTypes);
+        // add BDD to searchEntityTypes for Octane versions higher or eq than Coldplay P1 ( 15.1.4 - version where BDD was implemented )
+        OctaneVersion octaneVersion = OctaneVersionService.getOctaneVersion(connectionSettingsProvider.getConnectionSettings());
+        if (octaneVersion.isMoreOrEqThan(OctaneVersion.COLDPLAY_P1) && !searchEntityTypesCopy.contains(Entity.BDD_SCENARIO)) {
```

Comment:
why not use isLessThan, similar to prev PR

Reply:
in the prev PR, I had to remove it. now I added it. it was easier to implement this way